met-api/src/met_api/services/engagement_service.py

```diff
@@ -56,7 +56,8 @@ def get_engagement(self, engagement_id) -> EngagementSchema:
 
             engagement = EngagementSchema().dump(engagement_model)
             engagement['banner_url'] = self.object_storage.get_url(engagement['banner_filename'])
-        return engagement
+            return engagement
+        return None
```

Comment:
Why explicitly return `None`? This should not affect functionality; is this for legibility reasons?

Reply:
This is just for readability.